app/src/main/java/com/example/geoquiz/MainActivity.kt

```diff
@@ -5,30 +5,71 @@ import android.os.Bundle
 import android.util.Log
 import android.view.Gravity
 import android.widget.Button
+import android.widget.ImageButton
 import android.widget.TextView
 import android.widget.Toast
 
 class MainActivity : AppCompatActivity() {
     private lateinit var trueButton: Button
     private lateinit var falseButton: Button
+    private lateinit var nextButton: ImageButton
+    private lateinit var previousButton: ImageButton
+    private lateinit var questionTextView: TextView
+    private val questionBank = listOf(
+        Question(R.string.question_australia, true),
+        Question(R.string.question_oceans, true),
+        Question(R.string.question_africa, false),
+        Question(R.string.question_mideast, false),
+        Question(R.string.question_americas, true),
+        Question(R.string.question_asia, true)
+    )
+    private var currentIndex = 0
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         trueButton = findViewById(R.id.true_button)
         falseButton = findViewById(R.id.false_button)
+        nextButton = findViewById(R.id.next_button)
+        previousButton = findViewById(R.id.previous_button)
+        questionTextView = findViewById(R.id.question_text_view)
+
+
 
         trueButton.setOnClickListener {
-            // CHALLENGE: Customize the toast to show at the top instead of the bottom of the screen
```

Comment:
One small suggestion for later on when things will become crowded in the onCreate method. I like to create a separate `private fun setOnClickListeners()` function where I do all my on click listeners setup. This way I keep it nice and organised no matter how much code is added.